### ISSUE TITLE
School Scratch Project

### ISSUE THREAD

**Issue by LAnDo4567** [griffpatch/griffpatch.github.io#4]
Hey Griffpatch, I need an outside expert for a school project. I am working on a scratch project about global warming. Do you mind helping? Also, is there any other way to contact you?

**Comment by LankyBox01** [comment]
Yes, there is another way to contact him, you can just go to his youtube channel and comment in one of his posts! https://www.youtube.com/user/griffpatch

**Comment by sargent-coding** [comment]
Griffpatch probably has a lot of things on his hands. Try looking at a tutorial.

**Comment by MTM828** [comment]
> Hey Griffpatch, I need an outside expert for a school project. I am working on a scratch project about global warming. Do you mind helping? Also, is there any other way to contact you?

I can help but I was banned on Scratch.  >:[

**Comment by xItzDiamondx** [comment]
I can help.
